core/Db/Schema/Mysql.php

```diff
@@ -362,14 +362,14 @@ public function getTablesCreateSql()
             'changes'             => "CREATE TABLE `{$prefixTables}changes` (
                                       `idchange` INTEGER UNSIGNED NOT NULL AUTO_INCREMENT,
                                       `created_time` DATETIME NOT NULL,
-                                      `plugin_name` VARCHAR(255) NOT NULL,
+                                      `plugin_name` VARCHAR(40) NOT NULL,
```

Comment:
@sgiehl seems in other tables we have a 60 character limit in the DB like `plugin_setting`, `site_setting`. Does it make sense to apple this here as well? We could create a follow up issue for Matomo 5 (to not break anything) to adjust `isValidPluginName` to limit to max 60 characters. Checked our longest plugin name so far is around 30 characters.

Reply:
the developer docs say the max length should be 40. But we can also use 60. No preference there.

Reply:
OK we can leave it at 40. I will create a follow up issue then to change the other columns to 40 as well.

Reply:
or maybe let's use 60 @sgiehl for consistency with the other tables. I'm sure it be fine to use 40, but better have consistency for now. I created follow up issue for Matomo 5 to decide on a number.

Reply:
I've updated the plugin_name length to 60